.github/workflows/CI.yml

```diff
@@ -1,7 +1,24 @@
 on:
   push:
-    branches: [master]
+    branches:
+      - "master"
+    paths:
```

Comment:
changes to the ci config should also trigger a build

Reply:
True

Reply:
I think it should exclude changes to the book job manifest though.

Reply:
I've limited it to the CI.yml now